lib/services/notifications_service.rb

```diff
@@ -8,18 +8,26 @@ def initialize(http_service)
       @http_service = http_service
     end
 
-    def all(client_urn, params={page: 0, page_size: 25})
-      page      = params[:page]       || 0
-      page_size = params[:page_size]  || 25
+    def all(client_urn, params={page: 0, page_size: 25, auth_token: nil})
+      page       = params[:page]       || 0
```

Comment:
lining up the || is another thing you dont usually see in ruby code.  i saw this practice back in the .Net days.